analysis/trigger_performance/PHL_trigger_report.Rmd

```diff
@@ -0,0 +1,349 @@
+  margin-left: auto;
+  margin-right: auto;
+}
+</style>
+
+```{r README, echo = FALSE, message = FALSE, warning = FALSE}
+### NOTE: Performance metrics and their CIs should have been generated separately before the trigger report can be produced.
+
+### Create a trigger_report folder under DATA_DIR > /private/exploration/{country_pcode3/trigger_performance}
+### Create a "plots" folder sister to the RMD file in the country's `analyses` folder
+### Update the "hard-coded-values" code chunk below
+### If there are fewer or more than 3 triggers: Adjust the number of triggers and/or Packages throughout the Rmd (review the snapshot table and activation table carefully.)
+### Review, adjust or remove the country-specific code chunk
+### Knit
+```
+
+```{r hard-coded-values, echo = FALSE, message = FALSE, warning = FALSE}
+country_name <- "The Philippines"
+country_pcode3 <- "phl"
+shock <- "Typhoons"
+
+# trigger A info
+name_a <- "readiness" # what name is the trigger listed as in the perf_metrics_table.csv files?
+lead_time_a<- '4-7 days before landfall' # eg: 'Jan, Feb, Mar'
+data_source_a <- 'PAGASA, ECMWF, GEFS, Weather Nerds' # name of product + provider
+hist_period_a <- "2006 - 2021" # period for which historical data is available
+monitored_by_a <- 'OCHA Philippines' # owner of monitoring responsibilities
+
+# trigger B info
+name_b <- "activation"
+lead_time_b <- '72 hours before landfall'
+data_source_b <- 'ECMWF, 510 model'
+hist_period_b <- "2006 - 2021"
+monitored_by_b <- 'OCHA Philippines'
+
+# trigger C info
+name_c <- "framework"
+lead_time_c <- "N/A"
+data_source_c <- 'N/A'
+hist_period_c <- "N/A"
+monitored_by_c <- 'N/A'
+
+# Table footnotes
+trigger_table_footnote <- "The readiness and activation trigger can reach their threshold and activate independently from one another. The full framework activation is met if both the readiness and activation triggers are reachedt. The readiness trigger performance was assessed by comparing recommended activations with historical typhoon forecasts, and the activation trigger by running the 510 model on ECMWF hindcasts."
+#
+# # Data providers
+data_providers <- "OCHA Philippines, 510 Global"
+#
+# # Analysis
+# analysis <- "Historical data and analysis are available through IRI's Maproom [tool](http://iridl.ldeo.columbia.edu/fbfmaproom2/niger). The triggers were developed by IRI in close collaboration with DMN, WFP Niger, and the Centre for Humanitarian Data with valuable input from participating agencies and OCHA."
+#
+# Monitoring process
+monitoring_process <- "Between September and January, OCHA Philippines
+monitors typhoon forecasts from the Philippine Atmospheric, Geophysical
+and Astronomical Services Administration (PAGASA), the European Centre
+for Medium-Range Weather Forecasts (ECMWF), the Global Ensemble Forecast
+System (GEFS), and Weather Nerds (Private Initiative). If the typhoon
+is projected to impact areas in regions 5, 8 and 13 within 4-7 days, and exceed 178
```

Comment:
That's how they are specified in the framework, they tend to use the numbers instead of the names in the Philippines for some reason, but I think people understand where they are generally -- let me know if this addresses your concern.